apps/filters/views.py

```
@@ -0,0 +1,152 @@
+
+class FilterSetView(LoginAndTeamRequiredMixin, View):
+    """Handle PATCH (edit) and DELETE operations for FilterSet objects."""
+
+    def get_object(self, request, pk):
+        """Get the FilterSet object or return None if not found."""
+        try:
+            return FilterSet.objects.get(team=request.team, id=pk)
+        except FilterSet.DoesNotExist:
+            return None
+
+    def patch(self, request, team_slug: str, pk: int):
```

Ruff (0.14.0)

Unused method argument: `team_slug`

(ARG002)